apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryDataReader.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections;
+using System.Data;
+using System.Data.Common;
+
+namespace Google.Cloud.BigQuery.V2
+{
+    /// <summary>
+    /// </summary>
+    public sealed class BigQueryDataReader : DbDataReader
+    {
+#if NET45 || NET451
```

Comment:
Do we need to build both net45 and net451? If we're targeting net45, let's just use that in the `#if`.